operations/kubernetes/encryption/README.md

```diff
@@ -29,15 +30,52 @@ For Kubernetes secret encryption, once all control plane nodes agree on encrypti
 
 For further information, refer to the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/).
 
+## Setup
+
+The `encryption.sh` command will need to be made available on all control plane nodes beyond `ncn-m002`.
+Shown here for `ncn-m003`.
+
+1. (`ncn-m>002#`) Set the `ENCRYPTION_CMD_PATH` variable to `/usr/share/doc/csm/scripts/operations/kubernetes`.
```

Comment:
my thought is since we said the example is ncn-m003 -- that's what the prompt should be?  the `>` is throwing me off..

Reply:
Changed to ncn-m003.